--- a/cmake/pxrTargets-release.cmake
+++ b/cmake/pxrTargets-release.cmake
@@ -343,7 +343,7 @@ set_target_properties(hdStream PROPERTIES
   )
 
 list(APPEND _IMPORT_CHECK_TARGETS hdStream )
-list(APPEND _IMPORT_CHECK_FILES_FOR_hdStream "${_IMPORT_PREFIX}/plugin/usd/hdStream.lib" "${_IMPORT_PREFIX}/plugin/usd/hdStream.dll" )
+# list(APPEND _IMPORT_CHECK_FILES_FOR_hdStream "${_IMPORT_PREFIX}/plugin/usd/hdStream.lib" "${_IMPORT_PREFIX}/plugin/usd/hdStream.dll" )
 
 # Import target "usdImaging" for configuration "Release"
 set_property(TARGET usdImaging APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
